src/PersonObjects/Grafting/GraftableAugmentation.ts

```diff
@@ -18,6 +19,11 @@ export class GraftableAugmentation {
   }
 
   get cost(): number {
+    // Sell the Grafting-exclusive Aug at its starting price only
+    if (this.augmentation.name === AugmentationNames.CongruityImplant) {
+      return this.augmentation.startingCost;
```

Comment:
I'd rather you set the cost internally to 50t and let the formula handle the rest.